lnwallet/size.go

```diff
@@ -0,0 +1,117 @@
+package lnwallet
+
+import (
+	"github.com/roasbeef/btcd/blockchain"
+)
+
+const (
+	WitnessFactor = blockchain.WitnessScaleFactor
+	MaxTransactionCost = blockchain.MaxBlockCost
+
+	// Below you can see the structure of a Commitment transaction which
+	// will be broadcasted in case of force closure of channel. We should
+	// estimate its size rather than cooperative closure transaction,
+	// because this transaction is bigger.
+
+	// The weight(cost), which is different from the !size! (see BIP-141),
+	// is calculated as:
+	// Weight = 4 * BaseSize + WitnessSize (weight).
+	// BaseSize - size of the transaction without witness data (bytes).
+	// WitnessSize - witness size (bytes).
+	// Weight - the metric for determining the cost of the transaction.
+
+	// PW2SH:
+	//	- OP_0: 1 byte
+	//	- WitnessScriptSHA256: 32 bytes
+	P2SHSize = 1 + 32
+
+	// P2PKH:
+	//	- OP_0: 1 byte
+	//	- PublicKeyHASH160: 20 bytes
+	P2PKHSize = 1 + 20
```

Comment:
The variable should be named `P2WKHSize` as it's the fixed size of a v0 witness program.